internal/translator/data/languages_test.go

```diff
@@ -86,7 +86,7 @@ func TestLanguages(t *testing.T) {
 			t.Fatal(error)
 		}
 
-		t.Logf("Language found it %v", language)
+		t.Logf("Language updated, found it %v", language)
```

Comment:
Instead of print the  whole struct maybe the id :) 
```
t.Logf("Languate updated %s", language.ID)
```

Reply:
ok, no problem, I printed that because I had to view all the structure, but the suggestion was taken